server/routerlicious/packages/services-core/src/mongo.ts

```diff
@@ -4,59 +4,49 @@
  */
 
 import { Lumberjack } from "@fluidframework/server-services-telemetry";
-import { ICollection } from "./database";
+import { IDb, IDbFactory } from "./database";
 import { debug } from "./debug";
 
-export interface IDb {
-    close(): Promise<void>;
-
-    on(event: string, listener: (...args: any[]) => void);
-
-    collection<T>(name: string): ICollection<T>;
-}
-
-export interface IDbFactory {
-    connect(): Promise<IDb>;
-}
-
 /**
- * Helper class to manage access to a MongoDb database
+ * Helper class to manage access to database
+ * @TODO: Rename the file name as it behaves now as a generic DB Manager
```

Comment:
curious: what is the blocker for doing this rename in this PR? I think renaming to `DatabaseManager` or `DbManager` is a great idea

Reply:
Renaming the class and the file would make it harder to review the the core changes of this PR due to the number of changes. Additionally, it would increase the difficulty of rebase for other pending PRs after its merged.

My plan was to address renaming related changes in a dedicated PR. But if you still prefer to address all of them on this PR, I can include these changes too.